### DISCUSSION TITLE
2 boards one snake

### DISCUSSION THREAD

**Discussion by DAChenScratch** [BattlesnakeOfficial/feedback#22] [General]
2 boards 1 snake: Your snake is on 2 separate boards, with separate conditions, but only 1 request for a move will be sent. Both snakes on the 2 different board will both act to that  move. Food can be different positions on each board, and same for hazards. Snakes spawn in same position. Or maybe different idk

**Comment by andrewdmaclean** [comment]
Hey @DAChenScratch - just wanted to say thanks for the recommendation and to say that its on our list to look into 😄
